kani-driver/src/args_toml.rs

```diff
@@ -116,6 +136,14 @@ fn toml_to_args(tomldata: &str) -> Result<(Vec<OsString>, Vec<OsString>)> {
     Ok((args, cbmc_args))
 }
 
+fn unstable_entry(name: &String, value: &Value) -> Result<Option<OsString>> {
```

Comment:
I don't know what this function is doing.

Reply:
Let me add a comment. But this is parsing the new table `#[package.metadata.kani.unstable]` and converting it into `-Z <unstable_feature>` argument.